### ISSUE TITLE
Add inverse of `hasGeometry`

### ISSUE THREAD

**Issue by dr-shorthair** [opengeospatial/geosemantics-dwg#87]
Pointer from a geometry to the feature(s) that it is the geometry for - e.g. from GEOX

- [isGeometryOf](https://raw.githack.com/CSIRO-enviro-informatics/geosparql-ext-ont/master/geox.html#isgeometryof) 

Detail from #73

**Comment by akuckartz** [comment]
Which usecases exist for inverses ?

**Comment by dr-shorthair** [comment]
We have a Geometry Data Store which is managed separate from the Feature Store. 
The inverse property allows the source of the geometry in the store to be explicit. 

I know that some people consider inverse-properties to be an anti-pattern, because of synchronisation risks. And it is certainly true that the Geometry Store could just have the direct triples like`Feature99 `geo:hasGeometry Geometry88 .` stored alongside the geometries. But in this case it is the existence of the two stores that creates the risk, not the existence of the inverse predicate.

**Comment by akuckartz** [comment]
@dr-shorthair 
> We have a Geometry Data Store which is managed separate from the Feature Store.

That is an interesting and probably not unusual aspect of such architectures.

> And it is certainly true that the Geometry Store could just have the direct triples like Feature99 geo:hasGeometry Geometry88 . 

So it is as easy to not use the inverse triples? Or do they have any significant advantages?

**Comment by mathib** [comment]
@akuckartz : you mean not using `owl:inverseOf`, but a custom property (`ex:inverseProperyOf`) without reasoning consequences?

**Comment by akuckartz** [comment]
@mathib No, I do not understand why isGeometryOf is needed when triples like Feature99 geo:hasGeometry Geometry88 can be used (instead of Geometry88 geo:isGeometryOf  geo:Feature99).

**Comment by HolgerKnublauch** [comment]
Please don't add inverse properties. They create unnecessary complexity.
